conda/recipes/rmm/build.sh

```diff
@@ -1,13 +1,6 @@
-CMAKE_COMMON_VARIABLES=" -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_BUILD_TYPE=Release"
+# Copyright (c) 2018-2019, NVIDIA CORPORATION.
 
-# Cleanup local git
-git clean -xdff
-# Use CMake-based build procedure
-mkdir build
-cd build
-# configure
-cmake $CMAKE_COMMON_VARIABLES ..
-# build
-make -j${PARALLEL_LEVEL} VERBOSE=1 rmm_python_cffi
-# install
-make -j${PARALLEL_LEVEL} VERBOSE=1 rmm_install_python
+# This assumes the script is executed from the root of the repo directory
+printenv
+git clean -xdf
+./build.sh -v librmm
```

Comment:
This needs to be: 
```
./build.sh -v rmm
```
since this is the conda recipe for the `rmm` package

Reply:
Fixed